packages/react/src/ResourceDiffTable/ResourceDiffTable.module.css

```diff
@@ -10,6 +10,10 @@
   & td {
     padding: var(--mantine-spacing-sm) var(--mantine-spacing-sm);
     vertical-align: top;
+    word-wrap: break-word;
+    overflow-wrap: anywhere;
+    white-space: normal; 
+    min-width: 100px; 
```

Comment:
if a column is too wide. it makes sure the other do not wraps its content. Like in example B.